src/main/java/Counter.java

```diff
@@ -0,0 +1,11 @@
+public class Counter {
+    private int count;
+
+    public int getCount() {
+        return count++;
+    }
```

Comment:
Let's leave `getCount()` as a normal counter and make a separate `increment()` method to increase count value